src/Game/main.cpp

```diff
@@ -44,14 +46,18 @@ int main() {
     // Main loop.
     double targetFPS = 60.0;
     double lastTime = glfwGetTime();
-    double lastTimeRender = glfwGetTime();
-    while (!window->ShouldClose()) {
+    double lastTimeRender = glfwGetTime(); 
+    while (!window->ShouldClose() && numberOfFrames < 2400) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Condition 'numberOfFrames<2400' is always true [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AknownConditionTrueFalse)